src/main/java/com/example/car/sharing/controller/AuthenticationController.java

```diff
@@ -22,11 +26,14 @@ public class AuthenticationController {
     private final AuthenticationService authenticationService;
 
     @PostMapping("/login")
+    @Operation(summary = "Login user", description = "Login user with email and password, "
+            + "get JWT tokens")
     public UserLoginResponseDto login(@RequestBody @Valid UserLoginRequestDto request) {
         return authenticationService.authenticate(request);
     }
 
     @PostMapping(value = "/register")
+    @Operation(summary = "Register user", description = "Register user with user details")
```

Comment:
do u register any user? i think no and there can be nice to point that u register a new user!